docs/install.md

```diff
@@ -150,6 +138,122 @@ on the Kubernetes version.
+details.
+
+If the private keys and/or certificates data is missing in a deployment, the
+operator generates them using a self-signed CA.
+
+**NOTE:** A production deployment that is not supposed to depend on the 
+operator's self-signed CA instead must provide the certificates generated
+from a trusted certificate authority.
+
+- **Install the PMEM-CSI operator**
+
+``` console
+$ kubectl create -f https://github.com/intel/pmem-csi/raw/devel/deploy/operator/pmem-csi-operator.yaml
+```
+- **Create a custom resource**
+
+Once the [operator deployed](#deploy-PMEM-CSI-operator) is in `Running` state, it is ready to handle PMEM-CSI `Deployment` objects in `pmem-csi.intel.com` API group. Refer to [Deployment CRD API](#design.md#deployment-crd-api) for complete list of supported properties.
+
+Here is an example driver deployment by creating a custom resource:
+
+``` console
+$ kubectl create -f - <<EOF
+apiVersion: pmem-csi.intel.com/v1alpha1
+kind: Deployment
+metadata:
+  name: pmem-deployment
+spec:
+  pmemPercentage: 50
+  deviceMode: lvm
+  controllerResources:
+    requests:
+      cpu: "200m"
+      memory: "100Mi"
+  nodeResources:
+    requests:
+      cpu: "200m"
+      memory: "100Mi"
+EOF
+```
+
+Once the above deployment installation is successful, we can see all the driver
+pods in `Running` state:
+``` console
+$ kubectl get deployments.pmem-csi.intel.com
+NAME                 AGE
+pmem-deployment      50s
+
+$ kubectl describe deployment.pmem-csi.intel.com pmem-deployment
+Name:         pmem-deployment
+Namespace:    default
+Labels:       <none>
+Annotations:  <none>
+API Version:  pmem-csi.intel.com/v1alpha1
+Kind:         Deployment
+Metadata:
+  Creation Timestamp:  2020-01-23T13:40:32Z
+  Generation:          1
+  Resource Version:    3596387
+  Self Link:           /apis/pmem-csi.intel.com/v1alpha1/deployments/pmem-deployment
+  UID:                 454b5961-5aa2-41c3-b774-29fe932ae236
+Spec:
+  Controller Resources:
+    Requests:
+      Cpu:      200m
+      Memory:   100Mi
+  Device Mode:  lvm
+  Image:        localhost/pmem-csi-driver:canary
+  Node Resources:
+    Requests:
+      Cpu:     200m
+      Memory:  100Mi
+Status:
+  Phase:  Running
+Events:   <none>
+
+
+$ kubectl get po
+NAME                               READY   STATUS    RESTARTS   AGE
+pmem-deployment-controller-0       2/2     Running   0          51s
+pmem-deployment-node-4x7cv         2/2     Running   0          50s
+pmem-deployment-node-6grt6         2/2     Running   0          50s
+pmem-deployment-node-msgds         2/2     Running   0          51s
+pmem-csi-operator-749c7c7c69-k5k8n 1/1     Running   0          3m
+```
+
+#### Install from source
+
+- **Get source code**
+
+PMEM-CSI uses Go modules and thus can be checked out and (if that should be desired)
+built anywhere in the filesystem. Pre-built container images are available and thus
+users don't need to build from source, but they will still need some additional files.
```

Comment:
... but they may need some additional files for the following sections.

Reply:
Fixed.